hack/common.sh

```diff
@@ -56,6 +56,7 @@ readonly OPENSHIFT_BINARY_SYMLINKS=(
   openshift-sti-build
   openshift-docker-build
   openshift-gitserver
+  openshift-diagnostics
```

Comment:
If you're going to have this, I say shorter name.  At that length, it may as well just be `openshift diagnostics`.